Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -138,23 +138,20 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		m_Platform = DiscIO::IVolume::ELF_DOL;
 	}
 
-	if (!m_pImage.empty())
+	std::string path, name;
+	SplitPath(m_FileName, &path, &name, nullptr);
+	if (ReadPNGBanner(path + name + ".png"))
+	{ }
+	else if (ReadPNGBanner(path + "icon.png"))
+	{ }
```

Comment:
That's what ReadPNGBanner essentially is already. Also, IsPNGBannerPresent is a bad name because it doesn't indicate that it reads the PNG to m_Bitmap.